pkg/terraform/stages/azure/stages.go

```diff
@@ -7,26 +7,6 @@ import (
 	typesazure "github.com/openshift/installer/pkg/types/azure"
 )
 
-// PlatformStages are the stages to run to provision the infrastructure in Azure.
-var PlatformStages = []terraform.Stage{
-	stages.NewStage(
-		typesazure.Name,
-		"vnet",
-		[]providers.Provider{providers.AzureRM},
-	),
-	stages.NewStage(
-		typesazure.Name,
-		"bootstrap",
-		[]providers.Provider{providers.AzureRM, providers.Ignition, providers.Local},
```

Comment:
Azure is the only user of the `Local` terraform provider, so you could also `rm -rf terraform/providers/local`

Reply:
Will do. I am curious why it was there to begin with?

Reply:
I can't find any usage of it in the Azure terraform configs, so not sure... perhaps copypasta from a different provider. I think vsphere needed the local provider.

Reply:
After much frustration, it turns out, we in fact *do* use the local provider. It's used for "local_file".

Reply:
> After much frustration, it turns out, we in fact do use the local provider. It's used for "local_file".

Sorry for the frustration. What happened? I'm glad you spotted this, perhaps from the AzureStack CI failure? I limited my search to Azure. My bad.